packages/webpack/src/buildtime/policyGenerator.js

```diff
@@ -99,6 +100,14 @@ module.exports = {
         // Skip modules the user intentionally excludes.
         // This is policy generation so we don't need to protect ourselves from an attack where the module has a loader defined in the specifier.
         if (isExcludedUnsafe(module)) return
+        if (module.userRequest === undefined) {
+          diag.rawDebug(
+            1,
+            `LavaMoatPlugin: Module ${module} has no userRequest`
+          )
+          diag.rawDebug(2, { skippingInspectingModule: module })
```

Comment:
what is this option?

Reply:
It's just a console log based on the diagnostics level. The object is what gets logged